.github/workflows/images.yaml

```diff
@@ -9,17 +9,17 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v1
-      - uses: docker://docker.io/cilium/image-maker:8fb8f8325d966505552183f756eae4e3cb60d195
+      - uses: docker://quay.io/cilium/image-maker:8fb8f8325d966505552183f756eae4e3cb60d195
         name: Register binfmt from multi-platform builds
         with:
           entrypoint: docker
-          args: run --privileged linuxkit/binfmt:5d33e7346e79f9c13a73c6952669e47a53b063d4 
-      - uses: docker://docker.io/cilium/image-maker:8fb8f8325d966505552183f756eae4e3cb60d195
+          args: run --privileged linuxkit/binfmt:5d33e7346e79f9c13a73c6952669e47a53b063d4
```

Comment:
Damn, even took time to fix the trailing whitespace. What a champ! Thanks.

Reply:
`vim` does that automatically for mw on save :wink: